backend/app/api/endpoints/project_automations.py

```
@@ -0,0 +1,131 @@
+        db, project_id, automation_id, current_user.id
+    )
+
+
+@router.post(
+    "/{project_id}/automation-runs/{run_id}/cancel",
+    response_model=ProjectAutomationRunView,
+)
+def cancel_run(
+    project_id: str,
+    run_id: str,
+    db: Session = Depends(get_db),
```

Ruff (0.16.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)